app/views/admin/users/_reactions.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="row my-3">
+  <div class="col-12">
+    <h2 class="d-inline">Recent Reactions</h2>
+    <button type="button" data-toggle="collapse" data-target="#reactions-row" class="btn btn-secondary float-right">Toggle</button>
+  </div>
+  <div class="col-12 collapse" id="reactions-row">
+    <div class="list-group-flush">
+    <% unless @related_vomit_reactions.empty? %>
```

Comment:
Would it be worth it for me to use `counter_culture` here or should I stick to `.empty?` to follow the convention of the other partials like `_notes.html.erb`?

Reply:
I think `.empty?` is fine! I don't think it would make sense to use `counter_culture` here since you've already loaded these reactions already.